internal/services/automanage/automanage_configuration_profile_resource.go

```diff
@@ -0,0 +1,228 @@
+package automanage
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/hashicorp/go-azure-helpers/resourcemanager/commonschema"
+	"github.com/hashicorp/go-azure-helpers/resourcemanager/location"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/sdk"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/services/automanage/parse"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/services/automanage/validate"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tags"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/pluginsdk"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/validation"
+	"github.com/hashicorp/terraform-provider-azurerm/utils"
+	"github.com/tombuildsstuff/kermit/sdk/automanage/2022-05-04/automanage"
+)
+
+type AutoManageConfigurationProfileModel struct {
+	Name              string            `tfschema:"name"`
+	ResourceGroupName string            `tfschema:"resource_group_name"`
+	Configuration     string            `tfschema:"configuration_json"`
+	Location          string            `tfschema:"location"`
+	Tags              map[string]string `tfschema:"tags"`
+}
+
+type AutoManageConfigurationProfileResource struct{}
+
+var _ sdk.ResourceWithUpdate = AutoManageConfigurationProfileResource{}
+
+func (r AutoManageConfigurationProfileResource) ResourceType() string {
+	return "azurerm_automanage_configuration_profile"
```

Comment:
This should be
```suggestion
	return "azurerm_automanage_configuration"
```

Reply:
Thanks for the reply. Are you suggesting changing this resource name from azurerm_automanage_configuration_profile to azurerm_automanage_configuration? Cuz we have other related resource incoming, e.g. azurerm_automanage_configuration_profile_version. Shall I make this change to all of this series resources?

Reply:
@liuwuliuyun, yes profile would be removed from all the resource names

Reply:
Ok, let me remove this and resolve merge conflict.